### PR TITLE
Statics Managment

### LINKED ISSUE #35: Url routing issues
Home Url is <hostame>/web
weirdly when we click on add game the url become <hostname>/newGame (I expected it to be <hostname>/web/newGame)
Anyway, when we reload the page we have a 404, the same if we type directly the url
The same we the other urls

### LINKED ISSUE #41: React app urls can't be reached directly
Through the app, if we go to fisrt game url will be <hostname>/web/games/1
but if we type this directly into the browser url we got an error 500

In fact Gin search into assets the file games/1, which it can't find obviously

### PR DESCRIPTION
**Pull request by ggerbaud**
fix #41 
fix #35

### CODE REVIEW

**Comment by mathieu-pousse** [comment]
why do we have bedddbf6fc175301976c7e95711feb1664aaca60 included to this PR ?

File: server/middleware.go [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+	"github.com/gin-gonic/gin"
+	"github.com/gocaine/go-dart/server/autogen"
+	"io"
+	"mime"
+	"net/http"
+	"path/filepath"
+	"strings"
+)
+
+// RerouteToIndex returns a new middleware to handle not found paths (e.g. frontedn routes) and route them to index
```
Comment (mathieu-pousse) [inline server/middleware.go]: frontend

**Comment by mathieu-pousse** [comment]
why do we have that much changes in glide.lock ?

**Comment by ggerbaud** [comment]
Ok, so I removed unrelative commit, I'll put it in a new new PR.

I can't say for the glide.lock changes. I just compared with previous glide changes and it's quite the same.

**Comment by erwannT** [comment]
LGTM

**Comment by mathieu-pousse** [comment]
LGTM

**Pull request merged**